src/test/kotlin/blackjack/BlackjackManagerTest.kt

```diff
@@ -0,0 +1,25 @@
+package blackjack
+
+import blackjack.domain.BlackJackManager
+import blackjack.domain.CardDeck
+import blackjack.domain.Name
+import blackjack.domain.Player
+import blackjack.domain.Players
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+class BlackjackManagerTest {
+
+    @Test
+    fun `giveInitialCards()을 통해 cardDeck으로 부터 초기 2장의 카드를 나눠줄 수 있다`() {
```

Comment:
```suggestion
    fun `cardDeck으로 부터 초기 2장의 카드를 나눠줄 수 있다`() {
```
givenInitialCards()을 통해 라는 문구는 적어주지 않아도 될 것 같아요